lib/pf/pki_provider/scep.pm

```diff
@@ -0,0 +1,204 @@
+package pf::pki_provider::scep;
```

Comment:
Our package names are lowercase but the doc headers should say SCEP and not scep as we are talking about the protocol.

Fix all those in this file